aea/cli/fetch.py

```diff
@@ -49,10 +49,15 @@
 def fetch(click_context, public_id, alias, local, remote):
     """Fetch Agent from Registry."""
     ctx = cast(Context, click_context.obj)
+    is_mixed = not local and not remote
+    ctx.set_config("is_local", local)
```

Comment:
above you did a bit stricter: `local and not remote`...I guess better as `fetch` is importable and therefore consistency checks not guaranteed

Reply:
yes!

Reply:
given the `registry_flag` decorator, it shouldn't happen that both `local` and `remote` are both true, but of course that could change for some reason, so being stricter gives more consistency